Content.Client/UserInterface/ParticleAccelerator/ParticleAcceleratorControlMenu.cs

```diff
@@ -465,9 +465,11 @@ public PASegmentControl(ParticleAcceleratorControlMenu menu, IResourceCache cach
             {
                 _menu = menu;
                 _baseState = name;
-                _rsi = cache.GetResource<RSIResource>($"/Textures/Constructible/Power/PA/{name}.rsi").RSI;
+                _rsi = cache.GetResource<RSIResource>($"/Textures/Constructible/Specific/Engines/PA/{name}.rsi").RSI;
+                int _pos = name.LastIndexOf("_") + 1;
+                string _afterUnderscore = name.Substring(_pos, name.Length - _pos);
 
-                AddChild(_base = new TextureRect {Texture = _rsi[$"{name}c"].Frame0});
+                AddChild(_base = new TextureRect {Texture = _rsi[$"{_afterUnderscore}c"].Frame0});
```

Comment:
This is pretty dirty, you're better off passing it in to the ctor directly.

Also local variables are normally not prefixed with an underscore.